app/controllers/providers/proceeding_interrupts/related_orders_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Providers
+  module ProceedingInterrupts
+    class RelatedOrdersController < ProviderBaseController
+      before_action :proceeding
+      def show
+        @form = RelatedOrdersForm.new(model: proceeding)
+      end
+
+      def update
+        @form = RelatedOrdersForm.new(form_params)
+        return redirect_to providers_legal_aid_application_sca_interrupt_path(legal_aid_application, "plf_none_selected") if redirect_on_none_selected?
```

Comment:
Not a blocker: I find it slightly confusing that we are redirecting to a path in the SCA namespace for a PLF proceeding. Would it be worth moving the interrupts pages out of a specifically SCA namespace (could be a separate ticket)?

Reply:
AP-5607 has been created